cmd/fs-v1.go

```diff
@@ -266,19 +264,13 @@ func (fs *FSObjects) crawlAndGetDataUsageInfo(ctx context.Context, endCh <-chan
 			return filepath.SkipDir
 		}
 
-		path := strings.TrimPrefix(origPath, fs.fsPath)
-		path = strings.TrimPrefix(path, SlashSeparator)
-
-		splits := splitN(path, SlashSeparator, 2)
-		bucket := splits[0]
-		prefix := splits[1]
-
+		bucket, prefix := path2BucketObject(fs.fsPath, origPath)
 		if bucket == "" {
 			return nil
 		}
 
 		if isReservedOrInvalidBucket(bucket, false) {
-			return filepath.SkipDir
```

Comment:
filepath.SkipDir should help avoid a useless recursive listing of .minio.sys, is there any reason removing this ?

Reply:
In my tests this causes entire crawler to stop test with master on `minio server ~/`